static/src/js/zustand/slices/createProjectSlice.ts

```diff
@@ -265,11 +264,11 @@ const createProjectSlice: ImmerStateCreator<ProjectSlice> = (set, get) => ({
         const { data } = savedAccessConfigsResponse
         savedAccessConfigs = data
       } catch (error) {
-        reduxDispatch(actions.handleError({
-          error,
+        currentState.errors.handleError({
+          error: error as Error,
```

Comment:
I'm concerned about casting the error as an Error type in all these calls. Is it not already an error type? do we need to describe the type in our function better?

Reply:
Errors caught in try/catch blocks in TypeScript are of type `unknown` which is why I initially had the error type set as `unknown` in types.ts. So we either need to do these casts, or change the error type back to unknown.